src/browser/api_protocol/api_handlers/system.js

```diff
@@ -78,386 +73,303 @@ function SystemApiHandler() {
         return e !== undefined && e.constructor === Error;
     }
 
-    function setMinLogLevel(identity, message, ack, nack) {
-        const { payload: { level } } = message;
-        const response = System.setMinLogLevel(level);
-
-        if (didFail(response)) {
-            nack(response);
-
-        } else {
-            ack(_.clone(successAck));
-        }
+    function setMinLogLevel(identity, message) {
```

Comment:
Why did this become a promise?

Reply:
only way to nack is to return a rejected promise